pkg/v1/tkg/test/tkgctl/aws_cc/aws_cc_ccb_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
```

Comment:
I suggest renaming this file to ```aws_cc_custom_cb_test.go``` cause ```ccb``` doesn't help those unfamiliar with the code, what the the test file is about.

Reply:
changed file name to aws_cc_custom_cb_test.go